tensorflow/python/framework/ops.py

```diff
@@ -4399,7 +4400,7 @@ def _record_op_seen_by_control_dependencies(self, op):
     for controller in self._control_dependencies_stack:
       controller.add_op(op)
 
-  def control_dependencies(self, control_inputs):
+  def control_dependencies(self, *control_inputs):
```

Comment:
The docstring for control_inputs could use an update :)

Also, doesn't this break the behavior of `tf.control_dependencies(None)`?